containers/hydra/messager.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i python3 -p "python3.withPackages(ps: [ ps.slackclient ])"
+#
+# SPDX-FileCopyrightText: 2023 Technology Innovation Institute (TII)
+# SPDX-License-Identifier: Apache-2.0
+# ------------------------------------------------------------------------
+# Hydra Slack messaging script
+#
+# Prints given messaage to configured Slack channel with major hydra build
+# information data
+#
+# Slack configuration file format:
+# line1: Slack auth secret token (given when Slack application created)
+# line2: Used Slack channel (Slack application must be installed for this channel for usage)
```

Comment:
This should be documented also for the user (one just adding such configuration) in README.hydra.